### PR TITLE
Template distutils

### PR DESCRIPTION
**Pull request by Arignir**

### CODE REVIEW

File: stdlib/template/distutils.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    python_binary: str = 'python3',
+    script_path: str = './setup.py',
+    fail_ok: bool = True,
+):
+    """Run ``python3 ./setup.py build``.
+
+    :param args: Any extra arguments to give to ``setup.py``
+    :param python_binary: The command or path to use. The default value is ``python3``.
+    :param script_path: The path to the distutils script. The default value is ``./setup.py``.
+    :param fail_ok: If ``True``, the execution is aborted if ``cargo`` fails.
```
Comment (doom) [inline stdlib/template/distutils.py]: Cargo, really ?
Also, you document the default value as being `False`, but you set it to `True`

File: stdlib/template/distutils.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        This step installs the software in the install cache. The default value is :py:func:`.distutils_install`, with no argument.
+
+    **Split**
+
+        This step automatically splits the output of the build into multiple packages. The default value is :py:func:`~stdlib.split.drain_all.drain_all_with_doc`.
+        Alternative splitters can be found in the :py:mod:`~stdlib.split` module.
+
+    **Dependency Linking**
+
+        This step automatically finds requirements for the generated packages. The default value is :py:func:`~stdlib.deplinker.elf.elf_deplinker`.
```
Comment (doom) [inline stdlib/template/distutils.py]: Does that still make sense as a default for `distutils` ?

Comment (doom) [inline stdlib/template/distutils.py]: Does that still make sense as a default for `distutils` ?

File: stdlib/template/distutils.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+            packages = split()
+
+            if len(packages) > 0:
+                stdlib.log.ilog("The following packages were generated:")
+
+                with stdlib.log.pushlog():
+                    for package in packages.values():
+                        stdlib.log.ilog(str(package))
+
+    stdlib.log.ilog("Step 9/8: Dependency Linking")
```
Comment (doom) [inline stdlib/template/distutils.py]:
```suggestion
    stdlib.log.ilog("Step 8/8: Dependency Linking")
```

File: stdlib/template/distutils.py [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+"""
+
+import os
+import stdlib
+import stdlib.fetch
+import stdlib.extract
+import stdlib.patch
+import stdlib.split.drain_all
+import stdlib.deplinker.elf
+from multiprocessing import cpu_count
```
Comment (doom) [inline stdlib/template/distutils.py]: That's not useful here, you can remove it

File: stdlib/template/distutils.py (hunk 1 above)
Comment (Arignir) [reply stdlib/template/distutils.py]: Woops

**Pull request merged**